src/screens/Button.js

```diff
@@ -1022,6 +1022,28 @@ const ButtonPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="button.icon.size">
+          <Description>
+            Adjustments to the icon size on different screen sizes.
+          </Description>
+          <PropertyValue type="object">
+            <Example>{`icon: {
+              size: {
+                small: '12px',
+                medium: '18px',
+                large: '28px',
+              },
+            },`}</Example>
```

Comment:
Since we already specify `icon.size` in the property name we don't need to include those in the example
```suggestion
            <Example>{`{
              small: '12px',
              medium: '18px',
              large: '28px',
            }`}</Example>
```